packages/core/src/routes/hook.ts

```diff
@@ -46,7 +101,9 @@ export default function hookRoutes<T extends AuthedRouter>(
         params: { id },
       } = ctx.guard;
 
-      ctx.body = await findHookById(id);
+      ctx.body = await updateHookById(id, {
```

Comment:
just double check do we have a test that throws 404 if an invalid hook id is provided?

Reply:
Added in the integration tests.